.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ version: 2.1
 
 orbs:
   win: circleci/windows@2.2.0
-  # sfchangecase: salesforce/change-case-management@3
+  release-management: salesforce/npm-release-management@4.12.1
```

Comment:
Just like with source-deploy-retrieve, pin this version of npm-release-management in case there are breaking changes in one of the patch updates.

Reply:
Is it reasonable to expect that minor/patch won't have breaking changes?

Reply:
No, in the past when we started to use the CLI's orb we had some incidents where jobs/commands were renamed (as part of a minor or patch version bump) that broke us. This blocked us from being able to release.

I'm happy to change this though - it could be a good learning experience for everyone on just how important correct semver is.

Reply:
Gotcha. Let's leave it at `@4` since the documentation suggests that. If breaking changes happen again we can bring it up since that seems unintentional.